ios/CasperDash.xcodeproj/project.pbxproj

```diff
@@ -667,7 +667,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = CasperDash/CasperDash.entitlements;
 				CURRENT_PROJECT_VERSION = 23;
-				DEVELOPMENT_TEAM = 75YKDC6QCL;
+				DEVELOPMENT_TEAM = 77X6LN3CVN;
```

Comment:
Any reason why do we need to change this settings?

Reply:
Sorry, my mistake, because i test on my real device and i don't have apple developer team.